README.md

```diff
@@ -309,7 +309,7 @@ sudo eopkg install lazygit
 LAZYGIT_VERSION=$(curl -s "https://api.github.com/repos/jesseduffield/lazygit/releases/latest" | grep -Po '"tag_name": "v\K[^"]*')
 curl -Lo lazygit.tar.gz "https://github.com/jesseduffield/lazygit/releases/latest/download/lazygit_${LAZYGIT_VERSION}_Linux_x86_64.tar.gz"
 tar xf lazygit.tar.gz lazygit
-sudo install lazygit /usr/local/bin
+sudo mkdir -p /usr/local/bin/ && sudo install lazygit /usr/local/bin/
```

Comment:
I think a simpler form would be better:
```shell
sudo install lazygit -D -t /usr/local/bin
```
From `man install`:
```
-D     create all leading components of DEST except the last, or all components of --target-directory, then copy SOURCE to DEST
```

Reply:
Thanks for your advice!